react-app/src/features/package/package-status/index.tsx

```diff
@@ -13,34 +13,26 @@ export const PackageStatusCard: FC<{ id: string }> = ({ id }) => {
     <DetailCardWrapper title={"Status"}>
       <div className="my-3 max-w-2xl font-bold text-xl">
         <div>
-          {user?.isCms &&
-          !user.user?.["custom:cms-roles"].includes(UserRoles.HELPDESK)
+          {user?.isCms && !user.user?.["custom:cms-roles"].includes(UserRoles.HELPDESK)
             ? data?._source.cmsStatus
             : data?._source.stateStatus}
         </div>
         <div className="flex mt-1 flex-col gap-1 items-start">
-          {data._source.raiWithdrawEnabled && (
-            <div className="flex flex-row gap-1">
-              <p className="text-xs font-bold opacity-80">·</p>
-              <p className="text-xs opacity-80">
-                Withdraw Formal RAI Response - Enabled
-              </p>
-            </div>
-          )}
+          {data._source.raiWithdrawEnabled &&
+            data._source.seatoolStatus !== SEATOOL_STATUS.PENDING_APPROVAL &&
+            data._source.seatoolStatus !== SEATOOL_STATUS.PENDING_CONCURRENCE && (
```

Comment:
Maybe we can extract this logic into a function with a name that sorta explains the checks you're doing here?